Update footer links (#4529)
Couchers-org/couchers: app/web/markdown/plan.md

We are working on including the best parts from all the other hospitality exchange platforms out there. This includes hosting, surfing, messaging, live hangouts, planned events, trip planning, maps and many more.

If you have ideas for features you'd like to see, [please let us know](https://community.couchers.org/).

## Our plan

These are the key concepts that we are building into the foundation of the platform.

### [Non-profit structure](/plan/profit-and-incentives) <span class="tag tag-governance">Governance</span>

We are structured as a non-profit. There is no need for investors. We will explore various revenue models with the community.

### [Community-first with an improved verification system](/plan/communities-and-trust) <span class="tag tag-governance">Governance</span>

We prioritize communities by building them into the foundation of the platform. We open communication channels wherever we can. We will also incorporate an improved verification method to re-institute so members can better trust each other.

### [Member accountability](/plan/creeps-and-freeloaders) <span class="tag tag-design">Design</span>

> The community can be made safer by introducing accountability measures. If everyone is publicly accountable for their behavior, then the community will be able to filter people out if they are seen to be detrimental. This will be our first positive step in making the community safer, especially for women. This will also be used to filter freeloaders out of the platform.

Reviewer: ```suggestion
The community can be made safer by introducing accountability measures. If everyone is publicly accountable for their behavior, then the community will be able to filter people detrimental to the community. This is important to make the community safer, especially for women. This will also be used to filter freeloaders out of the platform.
```

### [Build it right](/plan/the-build) <span class="tag tag-tech">Technology</span>

> Our platform is developer-led, and we prioritizes building it well and making it scalable from the start. Having open communication channels with the broader community allows us to prioritize and tackle bugs and other issues.

Reviewer: ```suggestion
Our platform is developer-led, and we prioritize building it well and making it scalable from the start. Having open communication channels with the broader community allows us to prioritize and tackle bugs and other issues.
```
